lib/tether/interfaces.go

```diff
@@ -35,6 +36,7 @@ type Operations interface {
 	SetupFirewall() error
 	Apply(endpoint *NetworkEndpoint) error
 	MountLabel(ctx context.Context, label, target string) error
+	MountTarget(ctx context.Context, source url.URL, target string, mountOptions string) error
```

Comment:
Maybe we pass a struct here instead of this, which contains values we wanna pass to this function and that struct happens to have some methods on it to convert those values to the format that consumer consumes? (eg; MountOptions struct and String(), Source(), Target() methods))